Part_3/Dagim/bot/keyboards/keyboard.py

```diff
@@ -0,0 +1,20 @@
+
+from aiogram.types import ReplyKeyboardMarkup, InlineKeyboardMarkup, KeyboardButton, InlineKeyboardButton
+
+btn_menu_memes = KeyboardButton(text="Get-Memes")
+btn_menu_chat = KeyboardButton(text="Chat")
+btn_menu_show_profile = KeyboardButton(text="Show-Profile")
+
+
+btn_cancel = KeyboardButton(text="Cancel")
+btn_register_phone = KeyboardButton(text="register_phone", request_contact=True, )
+btn_register = KeyboardButton(text="Register")
+btn_inline_cancel = InlineKeyboardButton(text="Cancel", callback_data="handle_cancel",)
+btn_inline_show_profile = InlineKeyboardButton(text="Show Profile", callback_data="handle_show_profile",)
```

Comment:
The text at the function that handles show profile is text="Show-Profile" but here you added space instead of "-"

Reply:
so after registration the `Show Profile` inline is working for me, and

since it's inline-keyboard i don't think it will matter if  "-" is added, because i think it will only listen to the callback